node/core/runtime-api/src/lib.rs

```diff
@@ -396,10 +386,12 @@ where
 			query!(InboundHrmpChannelsContents, inbound_hrmp_channels_contents(id), sender),
 		Request::CurrentBabeEpoch(sender) => query!(CurrentBabeEpoch, current_epoch(), sender),
 		Request::FetchOnChainVotes(sender) => query!(FetchOnChainVotes, on_chain_votes(), sender),
-		Request::SubmitPvfCheckStatement(stmt, signature, sender) =>
-			query!(@NoCache, submit_pvf_check_statement(stmt, signature), sender),
-		Request::PvfsRequirePrecheck(sender) =>
-			query!(PvfsRequirePrecheck, pvfs_require_precheck(), sender),
+		Request::SubmitPvfCheckStatement(stmt, signature, sender) => {
+			query!(SubmitPvfCheckStatement, submit_pvf_check_statement(stmt, signature), sender)
```

Comment:
Can you elaborate on those changes?

Reply:
The fact that you don't cache the result is handled later in `store_cache`, returning `None` here would mean dropping the API request result. 
The pipeline is as follows:
1) In `query_cache` we propagate the request since it couldn't be cached
2) No essential changes in `make_runtime_api_request`
3) Do nothing in `store_cache`

The old behavior led to dropping the request immediately